apps/src/templates/instructions/TopInstructionsCSP.jsx

```diff
@@ -272,37 +278,57 @@ var TopInstructions = React.createClass({
             }
           </div>
         </PaneHeader>
-        {!this.state.helpTabSelected &&
-          <div style={[this.props.collapsed && commonStyles.hidden]}>
-            <div style={styles.body}>
-              {this.props.hasContainedLevels && <ContainedLevel ref="instructions"/>}
-              {!this.props.hasContainedLevels &&
-                <div ref="instructions">
+        <div style={[this.props.collapsed && commonStyles.hidden]}>
+          <div style={styles.body}>
+            {this.props.hasContainedLevels && <ContainedLevel ref="instructions"/>}
+            {!this.props.hasContainedLevels &&
+              <div ref="instructions">
+                {!this.state.helpTabSelected &&
                   <Instructions
                     ref="instructions"
                     renderedMarkdown={processMarkdown(this.props.markdown,
                       { renderer })}
                     onResize={this.adjustMaxNeededHeight}
                     inTopPane
                   />
-                  <TeacherOnlyMarkdown/>
-                </div>
-              }
-            </div>
-            {!this.props.isEmbedView &&
-              <HeightResizer
-                position={this.props.height}
-                onResize={this.handleHeightResize}
-              />}
-          </div>
-        }
-        {this.state.helpTabSelected &&
-          <div style={[this.props.collapsed && commonStyles.hidden]}>
-            <div style={styles.body}>
-              Help Tab
-            </div>
+                }
+                {this.state.helpTabSelected &&
+                  <div id="helpTab">
+                    <div className="reference_area" id="reference_area">
+                      <a
+                        className="video_link_rT"
+                        onClick={() => {
+                          showVideoDialog(
+                            {src: videoData.src,
+                            name: videoData.name,
+                            key: videoData.key,
+                            download: videoData.download,
+                            thumbnail: videoData.thumbnail,
+                            enable_fallback: true,
+                            autoplay: true}, true);
+                          }
+                        }
+                      >
+                        <img
+                          className="video_thumbnail_rT"
+                          src={videoData.thumbnail}
+                          alt={videoData.name}
+                        />
```

Comment:
instead of using className "video_thumbnail_rtT", can we just have inline styles now?